core/templates/components/summary-tile/lesson-card.component.ts

```diff
@@ -35,6 +35,7 @@ export class LessonCardComponent implements OnInit {
   @Input() story!: StorySummary | LearnerExplorationSummary | CollectionSummary;
   @Input() topic!: string;
   @Input() lastCard!: boolean;
+  @Input() completeCommunity?: boolean;
```

Comment:
Oh my apologies; I can see how it can be unclear. Currently the only way to measure any sort of progress for explorations (+ collections) is if they're retrieved from completedExplorationsList instead of incompleteExplorationsList (example - line 412 vs. 421 in community-lessons-tab.html). The variable is passed when for looping through them to display the correct progress values.

Reply:
Done